src/App.jsx

```diff
@@ -1,104 +1,28 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
```

Comment:
Ensure that the file './components/PostList/PostList' exists and exports a `PostList` component. If there are any issues with the file path or export, it could cause errors when rendering the component.